linker/linker_test.go

```diff
@@ -791,7 +791,7 @@ func TestLinkerValidation(t *testing.T) {
 				"foo.proto": `
 					syntax = "proto3";
 					import "google/protobuf/descriptor.proto";
-					enum Foo { true = 0; false = 1; t = 2; f = 3; True = 4; False = 5; inf = 6; nan = 7; }
+					enum Foo { option allow_alias = true; true = 0; false = 1; t = 2; f = 3; True = 0; False = 1; inf = 6; nan = 7; }
```

Comment:
Had to include a fix from https://github.com/jhump/protoreflect/pull/524/files#diff-f213d9afe67d61fc59d87d32853b67d02b40ad569e3c6d48620ccfe646a802dfR753 to get existing test passing.